docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -253,6 +253,10 @@ than `!equal?`. We also do not try to flag `eq` because if
 necessarily the same type as `b` since the `#==` operator can
 coerce objects for comparison.
 
+=== Safety
+
+This cop is unsafe because it changes how values are compared.
```

Comment:
A notable example is:
```ruby
Breed.new('Shepard').eql? Breed.new('Shepard') # => true
Breed.new('Shepard').equal? Breed.new('Shepard') # => false
```

Reply:
And the reasoning for making it unsafe is

> Integers, floats, booleans, symbols, and nil can be compared by identity and therefore the `be` matcher is preferable as it is a more strict test.

that we can't imply that the "actual" is one of those types that are safe to compare with `equal?`.